stetho/src/main/java/com/facebook/stetho/common/Util.java

```diff
@@ -23,6 +23,17 @@
     return item;
   }
 
+  public static <T> void throwIfNull(T item1, T item2) {
+    throwIfNull(item1);
+    throwIfNull(item2);
+  }
+
+  public static <T> void throwIfNull(T item1, T item2, T item3) {
```

Comment:
should we have multiple type arguments here, like so? (and for the 2 parameter version above of course)
```
public static <T1, T2, T3> void throwIfNull(T1 item1, T2 item2, T3 item3) { ...
```